test/test_jit.py

```diff
@@ -15182,6 +15182,42 @@ def forward(self, x):
         with self.assertRaisesRegex(RuntimeError, 'has no attribute'):
             torch.jit.script(wrapped)
 
+    def test_rescripting_loaded_modules(self):
+        class InnerSubmod(nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.register_buffer("foo", torch.ones(1))
+                self.register_parameter("bar", torch.nn.Parameter(torch.ones(1)))
+                self.baz = torch.ones(1)
```

Comment:
Would this be considered a constant or a data attribute? Just want to make sure we test constants because that code path has the most changes (e.g. constants are stored as `IValue`s and not `py::object`s now).

Reply:
forgot about constants, will add.